CHANGES.rst

```diff
@@ -6,6 +6,7 @@ Change log
 
 * Added ``-r`` option to ``purge_mail_log`` management command. Thanks julienc91
 * Fixed deprecation warnings on Django 3.1
+* added ability to override the default error handler
```

Comment:
Please make the sentence start uppercase (as the other two sentences above) and mention the name of the nmew setting `MAILER_ERROR_HANDLER` here. Thanks!